R/plot_metric.R

```diff
@@ -0,0 +1,17 @@
+#' Plot a line chart using ggplot
+#'
+#' @param metric A character vector with, at most, one element.
+#' @param date_from A character vector having the starting date, one element.
+#' @param date_to A character vector having the end date, one element.
+#'
+#' @return A ggplot line chart
+#' @export
+#'
+#' @examples
+#' x <- "positive_rate"
+#' plot_metric(x, date_from = "2022-01-01", "2022-01-15")

```

Comment:
```suggestion
#' plot_metric(x, date_from = "2022-01-01", date_to = "2022-01-15")

```
Better add date_to for easier illustration